weed/filer/rocksdb/rocksdb_store.go

```diff
@@ -11,17 +11,36 @@ import (
 	"github.com/chrislusf/seaweedfs/weed/glog"
 	"github.com/chrislusf/seaweedfs/weed/pb/filer_pb"
 	weed_util "github.com/chrislusf/seaweedfs/weed/util"
-	"github.com/tecbot/gorocksdb"
+	rocksdb "github.com/tecbot/gorocksdb"
```

Comment:
this is the same as the current package name. 

seems the go compiler is ok with this. But it is better to have a different name. The original name should be fine.